examples/vision/semantic_image_clustering.py

```diff
@@ -0,0 +1,577 @@
+"""
+Title: Semantic Image Clustering
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2021/02/28
+Last modified: 2021/02/28
+Description: Semantic Clustering by Adopting Nearest neighbors (SCAN) algorithm.
+"""
+
+"""
+## Introduction
+
+This example demonstrates how to apply the [Semantic Clustering by Adopting Nearest neighbors
+(SCAN)](https://arxiv.org/abs/2005.12320) algorithm, Van Gansbeke et. al, 2020, on the
```

Comment:
Suggestion:

- "et al." <- "et. al"
- Let's put round brackets around the author + year here (or em dashes); otherwise, it may look like you're demonstrating how to apply "the SCAN... algorithm, Van Gansbeke et al" (i.e. 2 things)

```suggestion
(SCAN)](https://arxiv.org/abs/2005.12320) algorithm (Van Gansbeke et al., 2020) on the
```